pkg/chunk/encoding/varbit_helpers.go

```diff
@@ -71,8 +71,5 @@ func isSignedIntN(i int64, n byte) bool {
 		return false
 	}
 	lower := upper - (1 << n)
-	if i < lower {
-		return false
-	}
-	return true
+	return i >= lower
```

Comment:
This file is copied from an older version of Prometheus; we do not modify such files unless critically necessary.

Reply:
I am sorry if this PR has caused any trouble, I only intend to help opensource projects adopt [DeepSource](https://deepsource.io). And no, I am not a bot :)